core/src/main/java/org/apache/accumulo/core/spi/cache/BlockCacheManager.java

```diff
@@ -24,6 +24,7 @@
 
 /**
  * @since 2.0.0
+ * @see org.apache.accumulo.core.spi
```

Comment:
I need to determine if this link works.  If so I want to add it more widely to SPI types.